src/app/Models/loggedInUser.ts

```diff
@@ -0,0 +1,9 @@
+export class LoggedInUser {
+  userName: string;
+  password: string;
+
+  constructor(username: string, password: string) {
+    this.userName = username;
+    this.password = password;
+  }
+}
```

Comment:
You can use interface instead of a class.

Reply:
updated thanks bhai.

Reply:
thanks bhai, updated